utils/auth.js

```diff
@@ -1 +1,23 @@
 const jwt = require('jsonwebtoken');
+<<<<<<< HEAD
+const secretKey = process.env.SECRET_KEY;
+
+const loginReq = async (req, res, next) => {
```

Comment:
try catch 대신, error.js의 catchAsync를 활용할 수 있습니다.
```suggestion
const loginReq = catchAsync(async (req, res, next) => {
```